perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -170,10 +170,15 @@ sub area_check {
     return 1 if FixMyStreet->staging_flag('skip_checks');
 
     my $councils = $params->{all_areas};
-    my $council_match = defined $councils->{$self->council_area_id};
-    if ($council_match) {
-        return 1;
+
+    # The majority of cobrands only cover a single area, but e.g. Northamptonshire
```

Comment:
This change presumably means the TfL specific version of this function can be dropped.